frontend/src/types/multimodal-live-types.ts

```
@@ -0,0 +1,242 @@
+  id: string;
+};
+
+/** Incoming types */
+
+export type LiveIncomingMessage =
+  | ToolCallCancellationMessage
+  | ToolCallMessage
+  | ServerContentMessage
+  | SetupCompleteMessage;
+
+export type SetupCompleteMessage = { setupComplete: {} };
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)